.github/workflows/selftest.yml

```diff
@@ -16,52 +16,54 @@ jobs:
     runs-on: ubuntu-latest
     name: Test run the e2e test on powerdns local testdata
     steps:
-      - name: checkout-repository
-        uses: actions/checkout@v2
-      - name: Setup go
-        uses: actions/setup-go@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
         with:
-          go-version: 1.18
+          go-version-file: go.mod
       - name: Run action for PowerDNS (exporter based)
         uses: ./
         with:
           spec_path: test/testdata/powerdns/powerdns-e2e.yml
           account_id: ${{ secrets.ACCOUNT_ID }}
           api_key: ${{ secrets.API_KEY }}
           license_key: ${{ secrets.LICENSE_KEY }}
+          retry_seconds: 60
+          retry_attempts: 15
   test_local_kafka:
     runs-on: ubuntu-latest
     name: Test run the e2e test on kafka local testdata
     steps:
-      - name: checkout-repository
-        uses: actions/checkout@v2
-      - name: Setup go
-        uses: actions/setup-go@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
         with:
-          go-version: 1.18
+          go-version-file: go.mod
       - name: Run action for Kafka (sample based)
         uses: ./
         with:
           spec_path: test/testdata/kafka/kafka-e2e.yml
           account_id: ${{ secrets.ACCOUNT_ID }}
           api_key: ${{ secrets.API_KEY }}
           license_key: ${{ secrets.LICENSE_KEY }}
+          retry_seconds: 60
+          retry_attempts: 15
```

Comment:
Would it make sense to use those values as [default values](https://github.com/newrelic/newrelic-integration-e2e-action/blob/main/main.go#L34)?

Reply:
Done in [#076efeb](https://github.com/newrelic/newrelic-integration-e2e-action/pull/25/commits/076efeb28476c4566b41a237ad99a3b292300311).

I modified also `action.yml` with the new defaults. I am ignoring the golangci-lint test as it was already as it is right now and it could lead to a myriad of cascade changes.